cmd/v0.22/pingsource-cleanup/README.md

```diff
@@ -0,0 +1,82 @@
+# PingSource cleanup post-0.22.0 install
+
+The following is a log of testing commands to try out this job.
+
+The job's log is the yaml of all resources that were deleted. The job uses
+eventing's service account.
```

Comment:
I think it would be safer to use a specific role. Here's an example of when we moved brokers (yes, it was pre-install, but same same but different):
https://github.com/knative/eventing/pull/3262/files

Reply:
@vaikas done